src/sentry/static/sentry/app/views/integrationInstallation.tsx

```diff
@@ -40,6 +41,29 @@ export default class IntegrationInstallation extends AsyncView<Props, State> {
     return t('Choose Installation Organization');
   }
 
+  trackOpened() {
+    const {organization} = this.state;
+    const provider = this.provider;
+    //should have these set but need to make TS happy
+    if (!organization || !provider) {
+      return;
+    }
+
+    trackIntegrationEvent(
+      {
+        eventKey: 'integrations.install_modal_opened',
+        eventName: 'Integrations: Install Modal Opened',
+        integration_type: 'first_party',
+        integration: provider.key,
+        //We actually don't know if it's installed but neither does the user in the view and multiple installs is possible
+        already_installed: false,
```

Comment:
I do wonder how much this will impact our analytics, I'm assuming not a ton but seems like something we should keep an eye on once this is merged, like what percentage of people installing GH are coming through the external flow.

Reply:
@MeredithAnya agreed, it will be interesting